hlx_statics/blocks/inlinealert/inlinealert.css

```
@@ -58,7 +58,7 @@ main div.inlinealert-wrapper .spectrum-InLineAlert-content {
     margin: 0;
     padding: 0;
     word-wrap: break-word;
-    color: rgb(44, 44, 44);
+    color: rgb(70,70,70);
```

Let's leave this unchanged. Given our background is either white or grey, darker is better for accessibility. It's ok to depart from Gatsby because we're trying to be more accessible in EDS

```suggestion
    color: rgb(44, 44, 44);
```